### PR TITLE
Pull request for automate-custom-line

### PR DESCRIPTION
**Pull request by fblackburn1**
POST-MERGE remove X-534

### CODE REVIEW

File: features/daily/line.feature [hunk 1]
```diff
@@ -14,3 +14,13 @@ Feature: Line
     Then I have the following hints:
       | exten        | line              |
       | 1603@default | PJSIP/newusername |
+
+  Scenario: Custom line
+    Given there are telephony users with infos:
+      | firstname | lastname | exten | context | protocol | with_phone |
+      | User      | SIP      | 1801  | default | sip      | yes        |
+      | User      | Custom   | 1802  | default | custom   | no        |
+    Given "User Custom" has custom interface equal to SIP interface from "User SIP"
```
Comment (pc-m) [inline features/daily/line.feature:23]: The interface could have been `Local/1801@default` to make the step implementation simpler

File: features/daily/line.feature [hunk 2]
```diff
@@ -14,3 +14,13 @@ Feature: Line
...
+    Given there is an incall "1802@from-extern" to the user "User Custom"
+    When incoming call received from "incall" to "1802@from-extern"
+    When "User SIP" is ringing
```
Comment (pc-m) [inline features/daily/line.feature:26]: Should be a `Then`

File: features/daily/line.feature (hunk 1 above)
Comment (fblackburn1) [reply features/daily/line.feature:23]: weeeeee :tada:

File: features/daily/line.feature (hunk 2 above)
Comment (fblackburn1) [reply features/daily/line.feature]: fixed

**Review by pc-m — APPROVED** [review]

**Comment by fblackburn1** [comment]
X-534 removed from testlink

**Pull request merged**